airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/spec.json

```diff
@@ -241,6 +248,9 @@
         "type": "object",
         "additionalProperties": false,
         "properties": {
+          "app_id": {
```

Comment:
The same property should be added to the `complete_oauth_server_output_specification` section.